libraries/lib-files/FileNames.cpp

```diff
@@ -45,7 +47,14 @@ used throughout Audacity into this one place.
 #include <windows.h>
 #endif
 
+#if defined(__WXGTK__)
+static wxString gOldUnixDataDir;
+#endif
+
+static wxString gConfigDir;
```

Comment:
I think we can have an array here by adding `Count` to  `DirTarget`:

```
enum class DirTarget
{
    Cache,
    Config,
    Data,
    State,

     Count
};

wxString directories[size_t(DirTarget::Count)];
```

Reply:
(Sorry for the formatting, GitHub aint a good text editor xD)

Reply:
Makes sense. Would `_targetCount` be an acceptable name? I would like to somehow distinguish that `Count` is a sentinel, not a valid Target.